src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -104,6 +104,7 @@ public function onKernelResponse(ResponseEvent $event)
             || ($response->headers->has('Content-Type') && false === strpos($response->headers->get('Content-Type'), 'html'))
             || 'html' !== $request->getRequestFormat()
             || false !== stripos($response->headers->get('Content-Disposition'), 'attachment;')
+            || ($response->headers->has('Symfony-Debug-Toolbar') && 'disable' === $response->headers->get('Symfony-Debug-Toolbar'))
```

Comment:
```suggestion
            || 'disable' === $response->headers->get('Symfony-Debug-Toolbar')
```

`HeaderBag::get()` should return `null` if the header is not present, so we can keep the code simple here.